core/tests/karma-globals.js

```diff
@@ -17,7 +17,21 @@
  */
 
 var GLOBALS = {
-  INTERACTION_SPECS: {},
+  INTERACTION_SPECS: {
+    TextInput: {
+      customization_arg_specs: [
+        {},
```

Comment:
INTERACTION_SPECS is *very* large, and autogenerated afaik. Do you know a way to replicate it without tons of manual copying? Maybe something with the js console? I couldn't figure it out.